### ISSUE TITLE
Cannot load whole playlist (over 200 items)

### ISSUE THREAD

**Issue by 4shadoww** [TimeTraveller-San/yTermPlayer#9]
Playlists with over 200 items cannot be loaded fully and that's because pafy's `get_playlist()` doesn't support more as it uses old version of the api. Pafy does have another way to get playlists `get_playlist2()` which could be used to solve the problem.

**Comment by 4shadoww** [comment]
I might do pr to fix this problem later today.

**Comment by 4shadoww** [comment]
get_playlist2() currently seems to have many problems so at it's current state this cannot be done. Example: mps-youtube/pafy#225. Waiting for next release.

**Comment by vaibkumr** [comment]
Yes, I noticed it a while ago. I tried get_playlist2() but it was crashing for some reason. It was a long while ago though. I guess we have to wait for the next release.
